source/release_notes.rst

```diff
@@ -1,14 +1,76 @@
 Release Notes
 =============
 
-The following release notes detail the updates to packages available on
-http://packages.fetchrobotics.com. For more details on the changes that
+The following release notes detail the updates to ROS Melodic packages
+maintained by Fetch Robotics and available at http://packages.ros.org.
+For more details on the changes that
 have occured in an individual package, please see the CHANGELOG within
 the installed package.
 
 For information on updating your robot to the latest packages, see
 :ref:`updating`.
 
+Note: For ROS Indigo, packages are hosted at http://packages.fetchrobotics.com.
+
+August 9, 2019
+--------------
+This release adds support for Fetch and Freight using ROS Melodic on Ubuntu 18.04.
+ROS Indigo and Ubuntu 14.04 are EOL. Robots using Indigo should continue to work,
+but will not receive further updates. To upgrade your robot, see
+`<indigo_to_melodic.rst>`__.
+
+ * fetch_moveit_config now includes an IKFast solver.
+ * fetch_moveit_config now includes apply_planning_scene plugin by default.
+ * fetch_teleop/bringup now includes teleop for the arm and cartesian controllers.
+ * fetch_bringup now includes a diagnostics aggregator.
+ * fetch/freight-system-config now preserves certain ethernet network configurations.
+
+New Fetch Packages:
+
+ * fetch-melodic-config:  0.2-0 (replaces fetch-system-config)
+ * freight-melodic-config:  0.2-0 (replaces freight-system-config)
+
+Updated Fetch Packages:
+
+ * ros-melodic-fetch-bringup: 0.7.5-0 -> 0.7.5-0
+ * ros-melodic-freight-bringup: 0.7.5-0 -> 0.7.5-0
+
+ # No release of the below? TODO all versions below
```

Comment:
no release of these? or no updates?

Reply:
You've got two similar lists, I bumped the middle number with the melodic release, so most are x.8.x

Reply:
As I mentioned offline, capturing loosely both releases this year. I haven't done the version numbers for the upcoming release as I wasn't sure which packages we'd actually release prior to talking to you today. (y)